website/project/model.py

```diff
@@ -1362,6 +1364,10 @@ def save(self, *args, **kwargs):
         else:
             suppress_log = False
 
+        self.root = self._root._id
```

Comment:
Oh, I see; at this point, the root node may not have been loaded. Thanks for the clarification.